### PR TITLE
[Junos]  NTP Resource Module Added

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### SUMMARY

```
tests/unit/modules/network/junos/test_junos_ntp_global.py                                91      0      0      0   100%
plugins/modules/junos_ntp_global.py                                                      16      1      2      1    89%
plugins/module_utils/network/junos/facts/ntp_global/ntp_global.py                       210     14    128     35    86%
plugins/module_utils/network/junos/config/ntp_global/ntp_global.py                      171      6    104     24    89%
```
 

##### ISSUE TYPE

 
- New Module Pull Request

##### COMPONENT NAME

junos_ntp_global
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by rohitthakur2590** [comment]
recheck

File: plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    "options": {
+                        "action": {
+                            "choices": ["accept", "reject"],
+                            "type": "str",
+                        },
+                        "value": {"type": "int"},
+                    },
+                    "type": "dict",
+                },
+                "trusted_keys": {"elements": "str", "type": "list"},
```
Comment (NilashishC) [inline plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py:109]: For compatibility with IOS argspec, this key has been made a list of dictionaries as follows in other platforms:
```
"trusted_keys": {
                    "type": "list",
                    "elements": "dict",
                    "no_log": False,
                    "options": {"key_id": {"type": "int"}},
                },
```

File: plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                        "peer": {"type": "str"},
+                        "prefer": {"type": "bool"},
+                        "version": {"type": "int"},
+                    },
+                    "type": "list",
+                },
+                "servers": {
+                    "elements": "dict",
+                    "options": {
+                        "key": {"type": "int"},
```
Comment (NilashishC) [inline plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py:83]: If this is just the key identifier, can we rename this to `key_id`?

Comment (rohitthakur2590) [reply plugins/module_utils/network/junos/argspec/ntp_global/ntp_global.py]: Yes, as other more platform has `key_id ` we can have same in Junos

File: meta/runtime.yml [hunk 3]
```diff
@@ -248,6 +252,8 @@ plugin_routing:
       redirect: junipernetworks.junos.junos_prefix_lists
     logging_global:
       redirect: junipernetworks.junos.junos_logging_global
+    ntp_global:
+      redirect: junipernetworks.junos.junos_logging_global
```
Comment (ashwini-mhatre) [inline meta/runtime.yml:256]: this should be junipernetworks.junos.junos_ntp_global

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Review by ashwini-mhatre — APPROVED** [review]

**Comment by rohitthakur2590** [comment]
recheck

**Pull request merged**